schemas/process.yml

```diff
@@ -61,9 +61,22 @@
         short_override: An array of previous executions for the process, including the initial fork. Only executable and args are set.
         normalize:
           - array
+      - at: process
+        as: responsible
+        short_override: Information about the responsible process.
+        beta: This field is beta and subject to change.
 
   fields:
 
+    - name: endpoint_security_client
+      level: extended
+      type: boolean
+      short: Indicates whether this process executable is an Endpoint Security client.
+      description: >
+          Processes that have an endpoint security client must have the com.apple.endpointsecurity entitlement and the value is set to true in the message.
+      example: true
```

Comment:
You don't need examples for boolean fields

Reply:
Thanks - i have removed the `example: true` keys.